### ISSUE TITLE
triclinic crystal system 

### ISSUE THREAD

**Issue by chtchelkatchev** [ICAMS/calphy#162]
Can calphy work with triclinic crystal system? For such a system, a "tri" thermostat in lammps is needed. I see only "iso" and "aniso" in the documentation...

**Comment by srmnitc** [comment]
Correct, at the moment the code in repo only has those two keywords. This is a straightforward implementation. I will mark it, and implement it.

**Comment by chtchelkatchev** [comment]
Thank You for quick answer!

**Comment by srmnitc** [comment]
Reopening just to keep track of implementation here.

**Comment by srmnitc** [comment]
Fixed in 1.7.2
